### ISSUE TITLE
Remove __id property from request sent using the blocks field type

### ISSUE THREAD

**Issue by roykoper** [sulu/sulu#5185]
| Q | A
| --- | ---
| Sulu Version | Sulu 2.0.4
| Browser Version | Google Chrome 80.0.3987.149

# Actual Behavior

How does Sulu behave at the moment? 

Odd behaviour when moving a block to the top, a new block gets created which can not be interacted(aside from moving it, then blocks start disappearing), I uploaded a video for clarification

# Expected Behavior

What is the behavior you expect?

I expect the block to get moved to the top without a new block appearing

# Steps to Reproduce

project-blocks.xml

```
<properties xmlns="http://schemas.sulu.io/template/template"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xmlns:xi="http://www.w3.org/2001/XInclude"
            xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">
    <block name="blocks" default-type="text" minOccurs="0">
        <meta>
            <title lang="en">Content</title>
        </meta>

        <types>
            <xi:include href="project-blocks/skill.xml"
                        xpointer="xmlns(sulu=http://schemas.sulu.io/template/template)xpointer(/sulu:properties/sulu:block/sulu:types/sulu:type)"/>

            <xi:include href="project-blocks/award.xml"
                        xpointer="xmlns(sulu=http://schemas.sulu.io/template/template)xpointer(/sulu:properties/sulu:block/sulu:types/sulu:type)"/>

            <xi:include href="project-blocks/job.xml"
                        xpointer="xmlns(sulu=http://schemas.sulu.io/template/template)xpointer(/sulu:properties/sulu:block/sulu:types/sulu:type)"/>

            <xi:include href="project-blocks/quote.xml"
                        xpointer="xmlns(sulu=http://schemas.sulu.io/template/template)xpointer(/sulu:properties/sulu:block/sulu:types/sulu:type)"/>

            <xi:include href="project-blocks/news.xml"
                        xpointer="xmlns(sulu=http://schemas.sulu.io/template/template)xpointer(/sulu:properties/sulu:block/sulu:types/sulu:type)"/>

        </types>
    </block>
</properties>
```

job.xml

```
<properties xmlns="http://schemas.sulu.io/template/template"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">
    <block name="project-blocks" default-type="job">
        <types>
            <type name="job">
                <meta>
                    <title>app.job</title>
                </meta>

                <properties>
                    <property name="id" type="single_job_selection">
                        <meta>
                            <title>app.job</title>
                        </meta>
                    </property>
                </properties>
            </type>
        </types>
    </block>
</properties>

```

skill.xml
```

<properties xmlns="http://schemas.sulu.io/template/template"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">
    <block name="project-blocks" default-type="skill">
        <types>
            <type name="skill">
                <meta>
                    <title>app.skill</title>
                </meta>

                <properties>
                    <property name="id" type="single_skill_selection">
                        <meta>
                            <title>app.projectSkill</title>
                        </meta>
                    </property>
                </properties>
            </type>
        </types>
    </block>
</properties>

```

I can post the other 3 xml files if needed

I am not quite sure if this a bug, or if I misconfigured something?

![block-bug](https://user-images.githubusercontent.com/39400442/78564507-d3706f80-781c-11ea-8f31-732a134d155e.gif)

**Comment by danrot** [comment]
It seems like this is caused by the internal `__id` attribute from the blocks. We also send this to the API, but other APIs ignore that and yours seem to store it and return it, causing this strange behavior. I think the fix would be to not send that `__id` to the server resp. return it from the field type.

**Comment by danrot** [comment]
Fixed in #5201
